redwood-tooling-codegen/src/main/kotlin/app/cash/redwood/tooling/codegen/testingGeneration.kt

```diff
@@ -426,6 +465,37 @@ internal fun generateWidgetValue(schema: Schema, widget: Widget): FileSpec {
             )
             .build(),
         )
+        .addFunction(
+          FunSpec.builder("toDebugString")
+            .addModifiers(OVERRIDE)
+            .returns(String::class)
+            .addCode(
+              buildCodeBlock {
```

Comment:
Instead of calling `.addCode` and then `buildCodeBlock` and then things like `addStatement`, you can call `apply` which will give you the ability to do `if`/`else` and call `addStatement` and the like